components/src/core/components/Input/DateInput.vue

```diff
@@ -194,6 +198,12 @@ export default defineComponent({
   },
 
   computed: {
+    isLengthyDate() {
+      return (
+        this.format === 'EEEE, dd-MMM-yyyy' ||
```

Comment:
Better to move this to a common place

Reply:
Moved to a by adding all the available date formats